provider/aws/formation/app.json.tmpl

```diff
@@ -80,7 +80,7 @@
         "Actions": [ { "Type": "forward", "TargetGroupArn": { "Ref": "Balancer{{ upper .Name }}TargetGroup{{ if .Internal }}Internal{{ end }}" } } ],
           "Conditions": [ { "Field": "host-header", "Values": [ { "Fn::Join": [ ".", [ "{{$.App}}-{{.Name}}", { "Fn::ImportValue": { "Fn::Sub": "${Rack}:{{ router .Name $.Manifest }}Host" } } ] ] } ] } ],
           "ListenerArn": { "Fn::ImportValue": { "Fn::Sub": "${Rack}:{{ router .Name $.Manifest }}Listener80" } },
-          "Priority": "{{ priority $.App .Name }}"
+          "Priority": "{{ priority $.App (print .Name $service) }}"
```

Comment:
.Name is already the name of the service here. Isn't this just creating duplication?

Reply:
You are right, will revert it.